python/pyspark/sql/streaming.py

```diff
@@ -786,9 +798,8 @@ def queryName(self, queryName):
         self._jwrite = self._jwrite.queryName(queryName)
         return self
 
-    @keyword_only
```

Comment:
Removed keyword_only, otherwise its weird if we have to write 
`writeStream.trigger(trigger=OneTime())`